### PR TITLE
README gets dedicated testing section

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

To help developers be able to consistently get tests to pass* locally.  (*all but one).

This PR is not "let's reorganize the README and improve it", but is meant as a surgical strike for "here's how to get tests running locally" which has been troubling for a number of developers.  I would ask that *this* PR be reviewed and approved if you think it accomplishes the goal, and additional README updates be ticketed separately (or just make a PR after this one is merged.)

If you want to see the updates formatted, visit https://github.com/sul-dlss/argo/tree/readme-testing

## How was this change tested?

3 of us "clumped" and ran through the steps manually

## Which documentation and/or configurations were updated?

README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,92 @@ bundle install
...
+           expected: 3
+                got: 2
+
+           (compared using ==)
+         # ./spec/helpers/items_helper_spec.rb:49:in `block (3 levels) in <top (required)>'
+
+    Finished in 0.80643 seconds (files took 3.43 seconds to load)
+    1 example, 1 failure
+    ```
+
```
Comment (mjgiarlo) [inline README.md:117]: I believe we can remove this now that we have https://github.com/sul-dlss/argo/pull/2119/files

Comment (mjgiarlo) [reply README.md:117]: #2119 has now been merged.

File: README.md [hunk 2]
```diff
@@ -187,24 +163,6 @@ docker-compose run --rm web bin/rake argo:repo:load
 
 ## Common tasks
 
-### Run the tests
```
Comment (mjgiarlo) [inline README.md:104]: @ndushay you OK with this change? I can restore if not. Didn't make sense to me to have testing covered in two spots.

Comment (ndushay) [reply README.md:104]: totally fine with me.

**Comment by mjgiarlo** [comment]
@ndushay feel free to merge if second commit is OK by you

File: README.md [hunk 3]
```diff
@@ -29,6 +29,68 @@ bundle install
 
 Note that `bundle install` may complain if MySQL isn't installed.  You can either comment out the `mysql2` inclusion in `Gemfile` and come back to it later (you can develop using `sqlite3`), or you can install MySQL.
 
+
+## Run the tests locally
```
Comment (jcoyne) [inline README.md:33]: Not a blocker, but you may want `docker-compose down` as this nukes the volumes for solr & fedora and allows you to start afresh.

**Pull request merged**